frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -130,3 +130,121 @@ Feature: Create the pipeline from builder page
+             Then user will be redirected to Pipeline Details page with header name "pipeline-when-expression"
+              And user will see tooltip saying "When expression" while scrolling over diamond structure before conditional task
+
+
+        @regression @to-do
+        Scenario: Code assistance for referencing params in the Pipeline Builder
+            Given user is at Pipeline Builder page
+             When user selects "git-clone" from Select task list
+              And user enters pipeline name as "pipeline-code-assistance"
+              And user clicks on Add Parameter
+              And user adds Name as "git-url"
+              And user Default value as "https://github.com/sclorg/nodejs-ex.git"
+              And user clicks on Add Workspace and add name as "git-workspace"
+              And user clicks on "git-clone" task node
+              And user enters url under Parameters section "$(params.git-url)"
+              And user adds workspace as "git-workspace"
+              And user clicks on Create
+             Then user will be redirected to Pipeline Details page with header name "pipeline-code-assistance"
+
+
+        @regression @to-do
+        Scenario: Code assistance for referencing workspaces in the Pipeline Builder
+            Given user has applied yaml "configMap-test-motd.yaml"
+            # user uses yaml content "using-optional-workspaces-in-when-expressions-pipelineRun/configMap-test-motd.yaml" in editor
+              And user is at YAML view of Pipeline Builder page
+             When user pastes the "pipelineRun-using-optional-workspaces-in-when-expressions.yaml" code
+            # user uses yaml content "using-optional-workspaces-in-when-expressions-pipelineRun/pipelineRun-using-optional-workspaces-in-when-expressions.yaml"
+              And user clicks on Create
+              And user clicks on Logs tab in PipelineRun details page
+             Then user will be able to see the output in print-motd task task
+
+
+
+        @regression
+        Scenario: Code assistance for referencing Context-based values in the Pipeline Builder
+            Given user is at pipelines page
+             When user clicks on import YAML button
+              And user pasted code of pipeline run "pipelineRun-using_context_variables.yaml"
+            # user uses yaml content "pipelineRun-using_context_variables.yaml"
+              And user clicks on Create
+              And user clicks on Logs tab in PipelineRun details page
+             Then user will be able to see the TaskRun UID, PipelineRun UID, Task name, TaskRun name, Pipeline name, PipelineRun name
+
+
+        @regression
+        Scenario: Code assistance for referencing Task Results in the Pipeline Builder
+            Given user has imported YAML "task-sum.yaml" and "task-multiply.yaml"
+            # user uses yaml content "sum-and-multiply-pipeline/task-sum.yaml" and "sum-and-multiply-pipeline/task-multiply.yaml" in editor
```

Comment:
May be you can mention the path of the file as well

Reply:
It's under test-data  and rest path I've mentioned